charts/internal/shoot-system-components/charts/ip-masq-agent/templates/daemonset.yaml

```diff
@@ -0,0 +1,50 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: ip-masq-agent
+  namespace: {{ .Release.Namespace }}
+spec:
+  selector:
+    matchLabels:
+      k8s-app: ip-masq-agent
+  template:
+    metadata:
+      labels:
+        k8s-app: ip-masq-agent
+    spec:
+      serviceAccount: ip-masq-agent
+      dnsPolicy: ClusterFirst
+      hostNetwork: true
+      priorityClassName: system-node-critical
+      tolerations:
+        - effect: NoSchedule
+          operator: Exists
+        - effect: NoExecute
+          operator: Exists
+        - key: CriticalAddonsOnly
+          operator: Exists
+      containers:
+        - args:
+            - --masq-chain=IP-MASQ
+            - --nomasq-all-reserved-ranges
+          image: {{ index .Values.images "ip-masq-agent" }}
+          imagePullPolicy: IfNotPresent
+          name: ip-masq-agent
+          securityContext:
+            privileged: true
+{{- if .Values.resources }}
+          resources:
+{{ toYaml .Values.resources | indent 12 }}
+{{- end }}
+          volumeMounts:
+            - mountPath: /etc/config
+              name: config
+      volumes:
+        - configMap:
```

Comment:
Where is this config coming from? I don't see it being created anywhere.

Reply:
its not, its optional later if live-config is required, have a look [here](https://github.com/kubernetes-sigs/ip-masq-agent) "As a result, the agent can be reconfigured in a live cluster by creating or editing this ConfigMap"

Reply:
OK, interesting. I think it's not obvious from the manifest itself, so probably a comment would help to understand what is going on with this configmap.